scripts/openmc-update-mgxs.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python
```

Comment:
Hopefully noone, but since we have a release out the door I cant assume thats true.